src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -505,22 +514,26 @@ export class SavedObjectsClient {
    * Add a saved object to workspaces
    * @param type
    * @param id
-   * @param workspaces
+   * @param options
    */
   addToWorkspaces = async <T = unknown>(
     type: string,
     id: string,
-    workspaces: string[]
+    options: SavedObjectsBaseOptions
```

Comment:
Discussed with @SuZhou-Joe, I may suggest to make the interface like this
```typescript
  deleteFromWorkspaces = async <T = unknown>(
    type: string,
    id: string,
    workspaces: string[],
    options: SavedObjectsBaseOptions = {}
  ) => {
```

Reply:
Nit: rename the workspaces: string[] to targetWorkspaces: string[] to reduce the confusion of duplicate workspaces params.

Reply:
Update.